package.json

```diff
@@ -23,9 +23,9 @@
   "bugs": "https://github.com/okta/samples-js-angular/issues",
   "homepage": "https://github.com/okta/samples-js-angular",
   "devDependencies": {
+    "dotenv": "^5.0.1",
```

Comment:
This dependency is needed to support the `testenv` file supported by TCK (https://github.com/okta/okta-oidc-tck/pull/32)

Reply:
Since we add this as a dependency, can we update the `CONTRIBUTING` file to show how to use the `testenv` file instead of:

```
export ISSUER=XYZ
```